index.html

```diff
@@ -0,0 +1,260 @@
+                  <p class="text-muted">9.19K subscribers</p>
+                </div>
+              </div>
+
+              <button>Subscribe</button>
+            </div>
+
+            <div class="links">
+              <div class="likes-and-dislikes">
+                <div class="like">
+                  <a href=""><i class="uil uil-thumbs-up"></i></a>
+                  <p>44</p>
+                </div>
+
+                <div class="vertical-line"></div>
+
+                <div class="dislike">
+                  <a href=""><i class="uil uil-thumbs-down"></i></a>
+                </div>
+              </div>
+
+              <div class="share">
+                <a href=""><i class="uil uil-share"></i></a>
+                <p>share</p>
+              </div>
+
+              <div class="more">
+                <a href=""><i class="uil uil-ellipsis-h"></i></a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <!-- MAIN ENDS -->
+
+      <!-- THUMBNAIL STARTS -->
+      <div class="thumbnail">
+        <div class="thumb-links">
+          <div class="link active">
+            <a href="#" class="active">All</a>
+          </div>
+          <div class="link">
+            <a href="#">From Microverse</a>
+          </div>
+          <span
+            ><a href="#"><i class="uil uil-angle-right-b"></i></a
+          ></span>
+        </div>
+
+        <div class="content">
+          <div class="content-details">
+            <div class="video">
+              <iframe
+                width="170"
+                height="95"
+                src="https://www.youtube.com/embed/qCdo-iqefDg"
+                title="YouTube video player"
+                frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                allowfullscreen
+              ></iframe>
+            </div>
+            <div class="details">
+              <a
+                href="https://www.youtube.com/watch?v=qCdo-iqefDg"
+                target="_blank"
+              >
+                <p class="details-title">
+                  How Lamia Landed a Remote Role In a US Based Tech Company
+                </p>
+              </a>
+              <p class="small">Microverse</p>
+              <div class="views">
+                <p>4k views</p>
+                <p>· 1 year ago</p>
+              </div>
+            </div>
+          </div>
+
+          <div class="content-details">
+            <div class="video">
+              <iframe
+                width="560"
+                height="315"
+                src="https://www.youtube.com/embed/8mceJBrgUkQ"
+                title="YouTube video player"
+                frameborder="0"
+                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+                allowfullscreen
+              ></iframe>
```

Comment:
I will suggest you use appropriate semantics. It will be great to use a **figure** and **figcaption**  tags to wrap the video and its title, respectively, to indicate that they are related content.